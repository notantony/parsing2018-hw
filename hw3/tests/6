ff aa bb cc =
= aa
        32
= bb
                                - aa 66

if    > a

            b
            print 32