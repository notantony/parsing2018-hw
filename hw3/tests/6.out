void ff(int aa, int bb, int cc) {
    aa = 32;
    bb = (aa - 66);
    if ((a > b))
        std::cout << 32;
}
